src/types/EnumPersonType.tsx

```diff
@@ -0,0 +1,5 @@
+export enum EnumPersonType {
```

Comment:
pls refactor your enum according to [enum naming convention](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase)